practice_app/search/views.py

```diff
@@ -28,3 +28,27 @@ def search(request):
     n = productList["Product"]
     return render(request, 'search/search.html', {"name": n, "key": "Search was found", "word": queryKeywords})
 
+def search2(request, slug):
```

Comment:
Name of the function should be more explanatory.

Reply:
Yes, you are right. Actually, I should delete it, it is a duplicate method.